pkgs/top-level/all-packages.nix

```diff
@@ -28435,6 +28435,8 @@ in
   serviio = callPackage ../servers/serviio {};
   selinux-python = callPackage ../os-specific/linux/selinux-python { };
 
+  signond = libsForQt5.callPackage ../development/libraries/signond {};
```

Comment:
This must be moved into `mkLibsForQt5`.

Reply:
Fixed in eb6bdaa89b6619bc9d18dba2d696f542f2b1fbda